src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2259,6 +2259,18 @@ private BoundExpression BindExplicitNullableCastFromNonNullable(ExpressionSyntax
             {
                 bag.Free();
             }
+
+            ErrorCode GetErrorCode(ConstantValue constantValue)
```

Comment:
Consider using a ternary and inlining: `var errorCode = constantValue == ConstantValue.Bad ? ErrorCode.ERR_ConstOutOfRangeChecked : ErrorCode.ERR_NoExplicitConv;`

nit: local functions are camelCased in compiler code